File: Aritra-Sasmal/MaxConsecutiveOnes.java
public class MaxConsecutiveOnes {
    public static int findMaxConsecutiveOnes(int[] nums) {
        int max = 0;
        int cnt = 0;

        for(int n: nums){
            cnt = (1+cnt)*n;
            max = Math.max(max,cnt);
        }
        return max;
    }
}

File: arun_lodhi/max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        i = 0
        j = 0
        for x in nums:
            if x==1:
                i = i+1
            else:
                if i>j:
                    j = i
                i =0
        if i>j:
            j = i
            return j
        else:
            return j
